src/Simplex/Chat/Store/Groups.hs

```diff
@@ -1361,7 +1364,8 @@ updateGroupProfile db User {userId} g@GroupInfo {groupId, localDisplayName, grou
         db
         "UPDATE groups SET local_display_name = ?, updated_at = ? WHERE user_id = ? AND group_id = ?"
         (ldn, currentTs, userId, groupId)
-      DB.execute db "DELETE FROM display_names WHERE local_display_name = ? AND user_id = ?" (localDisplayName, userId)
+      unlessM (checkLDNIsUser db user localDisplayName) $
+        DB.execute db "DELETE FROM display_names WHERE local_display_name = ? AND user_id = ?" (localDisplayName, userId)
```

Comment:
this can be one function that only deleted ldn if it's not a user

Reply:
DELETE FROM display_names
WHERE local_display_name = ? AND  user_id = ?
  AND local_display_name != (SELECT local_display_name FROM users where user_id = ?)